Resfeber/View Controllers/ContainerController.swift

```diff
@@ -13,35 +13,35 @@ class ContainerController: UIViewController {
     // MARK: - Properties
     
     fileprivate var sideMenuController: SideMenuController!
-    fileprivate var mainTabBarController: MainTabBarController!
+    fileprivate var mainNavigationController: UINavigationController!
     fileprivate let darkTintView = UIView()
     fileprivate var isExpanded = false
     
     // MARK: - Lifecycle
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        configureMainTabBarController()
+        configureMainNavigationController()
     }
     
     override var preferredStatusBarUpdateAnimation: UIStatusBarAnimation { .slide }
     override var prefersStatusBarHidden: Bool { isExpanded }
     
     // MARK: - Helpers
     
-    fileprivate func configureMainTabBarController() {
-        mainTabBarController = MainTabBarController()
-        mainTabBarController.sideMenuDelegate = self
-        
-        view.addSubview(mainTabBarController.view)
-        addChild(mainTabBarController)
-        mainTabBarController.didMove(toParent: self)
+    fileprivate func configureMainNavigationController() {
+        let exploreViewController = ExploreViewController()
```

Comment:
Eventually we will need to rename `ExploreViewController()` as that "Explore" feature is no longer planned for release. 

I suggest that it be named `TripsViewController()`.